runtime/compiler/control/CompilationThread.cpp

```diff
@@ -2946,6 +2948,10 @@ void TR::CompilationInfo::prepareForRestore()
    {
    J9JavaVM   *vm       = _jitConfig->javaVM;
    J9VMThread *vmThread = vm->internalVMFunctions->currentVMThread(vm);
+   TR::CompilationInfo * compInfo = getCompilationInfo(jitConfig);
```

Comment:
You are already in a member of CompilationInfo; you don't need to obtain another pointer to it